tests/unit/core/TestInstallAddon.js

```diff
@@ -110,10 +110,10 @@ describe('withInstallHelpers inner functions', () => {
         dispatch, { _addonManager: getFakeAddonManagerWrapper(), guid, installURL });
       return setCurrentStatus()
         .then(() => {
-          expect(dispatch.calledWith({
-            type: INSTALL_STATE,
-            payload: { guid, status: ENABLED, url: installURL },
-          })).toBeTruthy();
+          sinon.assert.calledWith(
```

Comment:
Yeah, the failure message makes it easier to solve. For example:

<img width="564" alt="screenshot 2017-06-05 22 22 33" src="https://cloud.githubusercontent.com/assets/55398/26812470/81b434c0-4a3d-11e7-92b0-ac2b427e226f.png">


We have an issue on file to convert everything eventually: https://github.com/mozilla/addons-frontend/issues/2437